packages/core/src/server/helper.ts

```diff
@@ -50,6 +50,14 @@ const formatPrefix = (prefix: string | undefined) => {
   return `${hasLeadingSlash ? '' : '/'}${prefix}${hasTailSlash ? '' : '/'}`;
 };
 
+export const pathParse = (publicPath: string): string => {
```

Comment:
Can be moved to `packages/core/src/helpers/path.ts`

We will move more helpers from `helper.ts` to `helpers/[name].ts` to avoid maintaining a large file.